azkaban-web-server/src/main/java/cloudflow/servlets/ExecutionServlet.java

```diff
@@ -54,10 +55,18 @@ private void handleAllExecutions(HttpServletRequest req, HttpServletResponse res
         executorService.getAllExecutions(req.getParameterMap()));
   }
 
-  private void handleSingleExecution(HttpServletRequest req, HttpServletResponse resp)
+  private void handleSingleExecution(HttpServletRequest req, HttpServletResponse resp, Map<String
+      , String> templateVariableMap)
       throws IOException {
-    // todo(sshardool): endpoint needs to be implemented
-    sendErrorResponse(resp, HttpServletResponse.SC_NOT_IMPLEMENTED, "not implemented");
+    String executionId = templateVariableMap.get(EXECUTION_ID_KEY);
+    ExecutionDetailedResponse executionResponse;
+    try {
+      executionResponse = executorService.getSingleExecution(executionId);
```

Comment:
Can you change the method name to `getExecution`? That would be more consistent with the naming pattern we've been using in the service methods.

Reply:
Not opposed to the change but would be good to have short discussion before we agree on a pattern.